meilisearch-release-changelog/v1.14.0.md

```diff
@@ -0,0 +1,137 @@
+```
+curl -H 'Content-Type: application/json' MEILISEARCH_URL/experimental-features -d '{"compositeEmbedders": true}'
+```
+2. Send a settings task (`PATCH /indexes/INDEX_UID/settings`) containing an embedder with `source: "composite"`, and with the parameters `searchEmbedder` and `indexingEmbedder` describing the embedder at search time (resp. indexing time). For example, using a [Hugging Face inference endpoint](https://huggingface.co/inference-endpoints/dedicated)
+```jsonc
+{
+    "embedders": {
+        "text": {
+            "source": "composite",
+            "searchEmbedder": {
+                "source": "huggingFace", // locally computed embeddings using a model from the Hugging Face Hub
+                "model": "baai/bge-base-en-v1.5",
+                "revision": "a5beb1e3e68b9ab74eb54cfd186867f64f240e1a"
+            },
+            "indexingEmbedder": {
+                "source": "rest", // remotely computed embeddings using Hugging Face inference endpoints
+                "url": "https://URL.endpoints.huggingface.cloud",
+                "apiKey": "hf_XXXXXXX",
+                "documentTemplate": "Your {{doc.template}}",
+                "request": {
+                    "inputs": [
+                        "{{text}}",
+                        "{{..}}"
+                    ]
+                },
+                "response": [
+                    "{{embedding}}",
+                    "{{..}}"
+                ]
+            }
+        }
+    }
+}
+```
+3. Send documents to the index. They will be indexed remotely using the Hugging Face inference endpoint.
+4. Perform semantic search queries, they will be embedded using the local model fetched from the Hugging Face Hub.
+
+For more details about this feature, please refer to its [public usage page](https://www.notion.so/meilisearch/Composite-embedder-usage-14a4b06b651f81859dc3df21e8cd02a0?pvs=4#19f4b06b651f809ab3ecc0daae7d5856)
+<TBD: Find a better title>
+
+Done by @dureuill in [#5371](https://github.com/meilisearch/meilisearch/pull/5371).
+
+## Other improvements
+
+* Get multiple documents by ids by @dureuill in [#5384](https://github.com/meilisearch/meilisearch/pull/5384)
+* Support merging update and replacement operations by @Kerollmops in [#5293](https://github.com/meilisearch/meilisearch/pull/5293)
+* Display the internal indexing steps with timings on the `/batches` route by @Kerollmops in [#5356](https://github.com/meilisearch/meilisearch/pull/5356)
+* exhaustive facet search by @ManyTheFish in [#5369](https://github.com/meilisearch/meilisearch/pull/5369)
+* Reduce RAM consumption of arroy by @irevoire in https://github.com/meilisearch/arroy/pull/105
+* Cache Embeddings in Search with an experimental feature by @dureuill in [#5418](https://github.com/meilisearch/meilisearch/pull/5418)
+* Extend the batch progress view to the indexing of vectors by @irevoire in [5420](https://github.com/meilisearch/meilisearch/pull/5420)
```

Comment:
Can you confirm there is no change in the API? I don't find anything in the PR and issue but the title made me doubt

Reply:
The contents of the `currentStep` field in batches is explicitly not stable and should be rendered as an object but as a `string`: https://www.notion.so/meilisearch/Enhance-visibility-on-batched-tasks-1194b06b651f810b8fe0fab5d72846a8?pvs=4#15a4b06b651f80e68c69efdf5ae8e043.

So this doesn't modify the API